static/redfish/v1/schema/OemLogEntry_v1.xml

```diff
@@ -33,6 +33,10 @@
               <Annotation Term="OData.LongDescription" String="A true value says external interface acked error log, false says otherwise."/>
             </Property>
 
+            <Property Name="GeneratorId" Type="Edm.String">
+              <Annotation Term="OData.Description" String="Id of the user who created the LogEntry."/>
```

Comment:
for dumps, client ip that has caused the dump creation will be assigned to this property and the bmcweb will then display it in the redfish response.

Likewise, whereever this prop is implemented, they must be assigning any unique id of the entity that has caused the operation and then should send it in the redfish response.
Hence i have given it as a string.

Your thoughts please?

Reply:
I think this should be a resource (e.g. a URI pointing at a session), or if it is an IP address it should be formatted that way, you probably need more properties for that. 

I think I am okay with this for eBMC.. 
We should revisit after Redfish gets this in the spec because a free-form string that can be multiple things "ip address, session id, client id." is a bad API design.

Reply:
>We should revisit after Redfish gets this in the spec because a free-form string that can be multiple things "ip address, session id, client id." is a bad API design.

Sure..